components/esp_lvgl_port/examples/i2c_oled/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+cmake_minimum_required(VERSION 3.16)
```

Comment:
I checked the CI and it seems that our `esp_lvgl_port/examples` applications are not built.

Could you please add a CI step to build them? Ideally using idf-build-apps